docs/docs/overview/quickstart/get-started-with-vanilla-js.md

```diff
@@ -0,0 +1,315 @@
+---
+sidebar_position: 8
+sidebar_label: Get started with vanilla JS
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Vanilla JS Quickstart
+
+Learn how to integrate Novu into your vanilla JS app on the fly. Send notifications across channels (SMS, Email, Chat, Push) with the help of a rich, customizable, real-time In-App notification center
```

Comment:
"Send notifications across channels (SMS, Email, Chat, Push) with the help of a rich, customizable, real-time In-App notification center." - This feels like  the notification center can send notifications across all these channels.

It can be reworded to something like:

Send notifications across different channels (SMS, Email, Chat, Push) and  enable real-time In-App notifications with the help of a rich and customizable Notification Center.

Reply:
I see your point. Making changes accordingly!